nautobot/users/factory.py

```diff
@@ -1,21 +1,40 @@
 from datetime import timezone
 
 from django.contrib.auth import get_user_model
+from django.urls import reverse
 import factory
 
-from nautobot.core.factory import BaseModelFactory, NautobotBoolIterator
+from nautobot.core.factory import BaseModelFactory, NautobotBoolIterator, random_instance, UniqueFaker
+from nautobot.users.models import SavedView
 
 User = get_user_model()
 
 
+def populate_saved_view_view_name():
+    from nautobot.extras.utils import FeatureQuery
+
+    VIEW_NAMES = []
+    # Append all saved view supported view names
+    for choice in FeatureQuery("saved_views").get_choices():
+        app_label, model = choice[0].split(".")
+        # Relevant list view name only
+        if app_label in ["circuits", "dcim", "ipam", "extras", "tenancy", "virtualization"]:
```

Comment:
Why is this check needed? Doesn't the FeatureQuery already restrict things appropriately?